modules/calib3d/src/stereobm.cpp

```diff
@@ -317,7 +317,9 @@ prefilterXSobel( const Mat& src, Mat& dst, int ftzero )
 }
 
 
-static const int DISPARITY_SHIFT = 4;
+static int DISPARITY_SHIFT;
```

Comment:
it's always a bad idea to use non-constant global variable; it makes the code thread-unsafe